detailed_docs/TRS/RC/GetInteriorVehicleDataConsent.md

```diff
@@ -2,23 +2,61 @@
 
 1.
 In case
-- SDL received OnRemoteControlSettings notification from HMI with "ASK_DRIVER" access mode
-- and RC application (in HMILevel FULL) requested access to remote control module that is already allocated to another RC application
-- and SDL requested user consent from HMI via GetInteriorVehicleDataConsent
-- and user allowed access to RC module for requested application
+- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
+- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
+- SDL doesn't have internally cached concent value
 
 SDL must
-- allocate access to RC module to requested application
-- process control request from this application
-- respond with resultCode SUCCESS
-- send OnRCStatus to HMI with this RC module in `allocatedModules`
-- send OnRCStatus to the application with this RC module in `allocatedModules` and with allowed:true
+- resend GetInteriorVehicleDataConsent to HMI
+- resend GetInteriorVehicleDataConsent response from HMI to mobile 
+- save driver's decision to cache
 
```

Comment:
@KhrystynaDubovyk Please Add requrements for next cases according last changes in GetInteriorVehicleDataConsent implementation:
 1.  
```
- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
- SDL received SetGlobalProperties request from mobile with userLocation which is within serviceArea of each requested in GetInteriorVehicleDataConsent modules or it is driver location. 
- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
- Requested modules are already allocated to another RC application
- Requested modules have allowMultipleAccess = true
- SDL has internally cached concent value
```
2.
```
- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
- SDL received SetGlobalProperties request from mobile with userLocation which out of serviceArea of each requested in GetInteriorVehicleDataConsent modules
- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
```
 3.  
```
- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
- SDL received SetGlobalProperties request from mobile with userLocation which is within serviceArea of each requested in GetInteriorVehicleDataConsent modules or it is driver location. 
- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
- Requested module is free.
```
 4.  
```
- SDL received OnRemoteControlSettings notification from HMI with `accessMode=ASK_DRIVER`
- SDL received SetGlobalProperties request from mobile with userLocation which is within serviceArea of each requested in GetInteriorVehicleDataConsent modules or it is driver location. 
- SDL received GetInteriorVehicleDataConsent request from mobile with valid `moduleIds`
- Requested modules are already allocated to another RC application
- Requested modules have allowMultipleAccess = false
```